Dockerfile uses alpine which is incompatible v1.16 and fails to build

https://pkgs.alpinelinux.org/package/edge/community/x86_64/zig is 0.16.0-r1

Dockerfile.omni uses zig 0.14

```
 > [20/22] RUN should_install() { case "zig" in ""|all|omni|"${1}") return 0 ;; *) return 1 ;; esac; };     if should_install zig; then       mkdir -p /work/tests       && cd /work       && /scripts/zig-setup.sh;     fi:
0.075 info: created src/root.zig
0.075 info: see `zig build --help` for a menu of options
0.079 Adding pylib-zig dependency...
0.660 /root/.cache/zig/tmp/b3d98dff2fad32c3/pylib-zig-main/build.zig.zon:11:14: error: expected string literal
0.666 Configuring build.zig with pylib-zig...
0.668 Creating placeholder main.zig...
0.669 Testing build setup...
5.744 /work/tests/build/common-zig-proj/build.zig:23:10: error: no field named 'root_module' in struct 'Build.ExecutableOptions'
5.744 /usr/lib/zig/std/Build.zig:639:31: note: struct declared here
5.756 Build setup failed!
------
Dockerfile:205
```

--- a/docker/Dockerfile
+++ b/docker/Dockerfile
@@ -14,9 +14,10 @@ ARG FLUTTER_VERSION=3.22.3
 ARG GO_VERSION=1.26.0
 ARG JULIA_VERSION=1.10.5
 ARG KOTLIN_VERSION=2.0.21
+ARG ZIG_VERSION=0.14.0
 
 ENV HOME=/root \
-    PATH="/root/.venv/bin:/root/go/bin:/usr/local/go/bin:/root/.cargo/bin:/root/.pub-cache/bin:/opt/v:/opt/flutter/bin:/opt/dart-sdk/bin:/opt/julia/bin:/opt/kotlinc/bin:${PATH}" \
+    PATH="/root/.venv/bin:/root/go/bin:/usr/local/go/bin:/root/.cargo/bin:/root/.pub-cache/bin:/opt/v:/opt/flutter/bin:/opt/dart-sdk/bin:/opt/julia/bin:/opt/kotlinc/bin:/opt/zig:${PATH}" \
     CXX=clang++ \
     CXXFLAGS="-fuse-ld=lld" \
     CLANG_FORMAT_STYLE=Google \
@@ -57,8 +58,8 @@ RUN apk add --no-cache \
       xz \
       z3 \
       z3-dev \
-      zig \
-      zip
+      zip \
+      zstd
 
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
@@ -204,7 +205,14 @@ RUN should_install() { case "${FOCUS}" in ""|all|omni|"${1}") return 0 ;; *) ret
 
 RUN should_install() { case "${FOCUS}" in ""|all|omni|"${1}") return 0 ;; *) return 1 ;; esac; }; \
     if should_install zig; then \
-      mkdir -p /work/tests \
+      arch="$(apk --print-arch)" \
+      && curl -fsSL -o /tmp/zig.tar.xz \
+        "https://ziglang.org/download/${ZIG_VERSION}/zig-linux-${arch}-${ZIG_VERSION}.tar.xz" \
+      && mkdir -p /opt/zig \
+      && tar -xJf /tmp/zig.tar.xz -C /opt/zig --strip-components=1 \
+      && rm /tmp/zig.tar.xz \
+      && zig version \
+      && mkdir -p /work/tests \
       && cd /work \
       && /scripts/zig-setup.sh; \
     fi

--- a/pyzig/transpiler.py
+++ b/pyzig/transpiler.py
@@ -366,15 +366,19 @@ class ZigTranspiler(CLikeTranspiler):
         return f'const {name} = @import ("{name}");'
 
     def _import_from(self, module_name: str, names: List[str], level: int = 0) -> str:
-        names = ", ".join(names)
         if len(names) == 1:
             # TODO: make this more generic so it works for len(names) > 1
             name = names[0]
             lookup = f"{module_name}.{name}"
             if lookup in MODULE_DISPATCH_TABLE:
                 zig_module_name, zig_name = MODULE_DISPATCH_TABLE[lookup]
-                return f"from {zig_module_name} import {zig_name}"
-        return f"from {module_name} import {names}"
+                return (
+                    f'const {zig_name} = @import("{zig_module_name}.zig").{zig_name};'
+                )
+        lines = []
+        for name in names:
+            lines.append(f'const {name} = @import("{module_name}.zig").{name};')
+        return "\n".join(lines)
 
     def visit_List(self, node) -> str:
         self._headers.add("pylib")

--- a/tests/dir_cases/test1-zig-expected/bar.zig
+++ b/tests/dir_cases/test1-zig-expected/bar.zig
@@ -1,0 +1,4 @@
+const std = @import("std");
+pub fn bar1() i32 {
+    return 0;
+}

--- a/tests/dir_cases/test1-zig-expected/baz.zig
+++ b/tests/dir_cases/test1-zig-expected/baz.zig
@@ -1,0 +1,4 @@
+const std = @import("std");
+pub fn baz1() []const u8 {
+    return "foo";
+}

--- a/tests/dir_cases/test1-zig-expected/foo.zig
+++ b/tests/dir_cases/test1-zig-expected/foo.zig
@@ -1,0 +1,10 @@
+const std = @import("std");
+const expect = std.testing.expect;
+const bar1 = @import("bar.zig").bar1;
+const baz1 = @import("baz.zig").baz1;
+pub fn main() !void {
+    const x: i32 = bar1();
+    const y: []const u8 = baz1();
+    try expect(x == 0);
+    try expect(y == "foo");
+}

